src/en/news/blog/2022/bucket-notifications-with-knative-and-rook-on-minikube2/index.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "Bucket Notifications in Rook - Part II"
+date: "2022-03-16"
+author: "ylifshit"
+---
+
+## Why?
+
+[Last time](https://ceph.io/en/news/blog/2021/bucket-notifications-with-knative-and-rook-on-minikube/) we saw how to use bucket notification together with Knative on minikube.
+Most of the process was "easy as a YAML" - most, but not all...
+
+In order to setup the bucket notifications part of the solution some manual steps were neeeded.
+However, as proimised, a set of [new CRs](https://rook.io/docs/rook/v1.8/ceph-object-bucket-notifications.html) has come to our rescue in Rook1.8!
+
+Now the entire process could be done using YAMLs only.
+
+## The "Moving Parts"
+
+We are going to use the same infrastructure bits as in the previous post (with some improvments): minikube, Knative and Rook
+
+### minikube
```

Comment:
What about simplifying the blog by assuming they already have a K8s cluster? Or if they don't you could link to the rook topic on [starting minikube](https://rook.github.io/docs/rook/v1.8/development-environment.html#minikube)? Then the blog could focus more on the object storage and bucket usage.

Reply:
i wanted the blog to be "self contained" - but probably reference the rook page as well, in case things are being updated